README.md

```diff
@@ -257,6 +257,26 @@ These additional options are specific to pug templates and can be configured in
 - `pugEmptyAttributesForceQuotes`  
   Define a list of patterns for attributes that will be forced to have empty quotes even with "none" selected.
 
+- `pugClassNotation`  
+  Define how classes should be formatted.
+
+  Choices:
+
+  - `'as-is'` -> Disables class formatting.  
+    Example: `foo.bar(class="baz")`
+  - `'literal'` _default_ -> Forces all valid classes to be printed as literals.  
```

Comment:
Could you swap these options, so the default will placed first
Also make a comment, that the third option `attributes` will come later (could we style this in any way? like grey-ish text-color?)